src/app/services/get-data/get-data.service.ts

```diff
@@ -25,13 +25,19 @@ export class GetDataService {
   private glossaryArticleURL = "https://build-oracc.museum.upenn.edu/neo/";
   private baseUrl = "https://build-oracc.museum.upenn.edu/";
   private sourceURL = "http://cdli.ucla.edu/";
+  private searchSuggestionsUrl =
+    "https://build-oracc.museum.upenn.edu:5000/suggest_all/";
```

Comment:
It's safe to assume that all the endpoints will be available on the same server. We currently define the server URL 4 times, could instead store it in a `const` and interpolate it in the other variables?

Reply:
Improved with latest commit